### DISCUSSION TITLE
new firmware for RLC-520A V2

### DISCUSSION THREAD

**Discussion by FGOD1983** [AT0myks/reolink-fw-archive#35] [Firmware sharing]
I bought myself a RLC-520A V2 last week which came with newer firmware, after some issues support asked me to downgrade, but as I wanted to go back to the firmware it came with, I was unable to as I couldn't find it anywhere...

I asked support for the firmware and they sent me this link: https://support-d.reolink.com/attachments/token/b32M4tyle4rDpYdNSATF25tZP/?name=IPC_523128M5MP_V2.2368_23062701.RLC-520A.OV05A10.5MP.REOLINK.pak

You can add this to the list of RLC-520A V2 firmwares with a link to the Reolink support forum where I posted about it: community.reolink.com/post/26219

**Comment by AT0myks** [comment]
Thanks, this firmware has recently been added in 3ec625a6d1a6e413d5ef23f5977799bb3e6942cc (see #25) but with a different link so I added yours as an alternative in afd6ac08cc9a8de799dd07fa4ed315829036fb0d along with your post as another source.

**Comment by FGOD1983** [comment]
No problem! Must have missed the other one while I was looking for it 😅 but it is never bad to have backup links 😁
